ADOPTERS.md

```diff
@@ -16,7 +18,11 @@ _Add yours here!_
 
 Below is a list of solutions where Contour is being used as a component.
 
-**[VMware](https://tanzu.vmware.com/tanzu)**  
+**[Knative](https://knative.dev)**
+
+Knative can use Contour to serve all incoming traffic via the `net-contour` ingress Gateway. The net-contour controller enables Contour to satisfy the networking needs of Knative Serving by bridging Knative's KIngress resources to Contour's HTTPProxy resources.
```

Comment:
nit: Maybe link out to the net-contour repo? (https://github.com/knative-sandbox/net-contour)

Reply:
Yes, of course!! Thanks for thinking of this, changing..